modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2484,6 +2491,15 @@
 		return m.conditionFromConfigText(vstudio.projectConfig(cfg))
 	end
 
+--
+-- Get the latest installed SDK 10 version from the registry.
+--
+
+	function m.latestSDK10Version()
+		local arch = iif(os.is64bit(), "\\WOW6432Node\\", "\\")
+		local version = os.getWindowsRegistry("HKLM:SOFTWARE" .. arch .."Microsoft\\Microsoft SDKs\\Windows\\v10.0\\ProductVersion")
+		return iif(version ~= nil, version .. ".0", nil)
```

Comment:
This took me a little bit to figure out but these tests fail on Linux and macOS because `version .. ".0"` is being evaluated here. To resolve this issue, you'll need to break this out into a proper if statement.